src/main/kotlin/exercise2/task5/CreateUserDSL.kt

```diff
@@ -49,11 +49,16 @@ internal data class Address(
  */
 
 internal fun user(initUser: User.() -> Unit): User {
-    TODO("Implement me!!!")
+    val newUser = User()
```

Comment:
To je to, bravo! 

Potencijalno unapređenje ovog koda bi bilo da se koriste scope funkcije: 
```suggestion
    internal fun user(initUser: User.() -> Unit): User = User().apply(initUser)
```